readthedocs/oauth/constants.py

```diff
@@ -0,0 +1,9 @@
+GITHUB_BRAND = 'GitHub'
+GITLAB_BRAND = 'GitLab'
+BITBUCKET_BRAND = 'Bitbucket'
+
+VCS_PROVIDER_CHOICES = (
+    (GITHUB_BRAND, 'GitHub'),
```

Comment:
Shouldn't this be `(slug, display_name)`, like `('github', 'GitHub')` instead?

Reply:
Yes, you are right, but I have intensionally added it like this because this requires the constants to be in this camel case (``'GitHub'``)

https://github.com/readthedocs/readthedocs.org/blob/811890894f72c14be05ddd443a20adc017656c5c/readthedocs/projects/tasks.py#L1916

@humitos I would like to know your thoughts on this.

Reply:
Good catch! I _personally_ think `*_BRAND` should only be used when we need to show them to users and internally, we should use a slug.

Reply:
Then we need to keep them separated, My original intention to merge the both was to not make it confusing in the future as they are almost the same. But I get your point. :)